### PR TITLE
Add Cocoapods support

### PR DESCRIPTION
**Pull request by NSMyself**
I took the liberty of adding Cocoapods support, hope you don't mind.

There's still a ton of stuff that can be configured, particularly the `tag` field. 
This would imply that you add a tag to a particular commit I believe.

### CODE REVIEW

File: Receiver.podspec [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  These will help people to find your library, and whilst it
+  #  can feel like a chore to fill in it's definitely to your advantage. The
+  #  summary should be tweet-length, and the description more in depth.
+  #
+
+  s.name         = "Receiver"
+  s.version      = "0.0.1"
+  s.summary      = "Receiver is nothing more than an opinionated micro framework implementation of the Observer pattern (~120 LOC)"
```
Comment (RuiAAPeres) [inline Receiver.podspec]: the summary should be `Swift µframework implementing the Observer pattern`

File: Receiver.podspec [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+  # This description is used to generate tags and improve search results.
+  #   * Think: What does it do? Why did you write it? What is the focus?
+  #   * Try to keep it short, snappy and to the point.
+  #   * Write the description between the DESC delimiters below.
+  #   * Finally, don't worry about the indent, CocoaPods strips it!
+  s.description  = <<-DESC
```
Comment (RuiAAPeres) [inline Receiver.podspec]: On the description you would add the summary you put on top.

File: Receiver.podspec [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  #  If this Pod runs only on iOS or OS X, then specify the platform and
+  #  the deployment target. You can optionally include the target after the platform.
+  #
+
+  # s.platform     = :ios
+  # s.platform     = :ios, "5.0"
+
+  #  When using multiple platforms
+  s.ios.deployment_target = '10.0'
+  s.tvos.deployment_target = '10.0'
```
Comment (RuiAAPeres) [inline Receiver.podspec]: I have yet to create a target for `tvos`, although I think it would work.

Comment (NSMyself) [reply Receiver.podspec]: I'll comment it for now

**Comment by RuiAAPeres** [comment]
Damn, forget to ask you to remove the comments.

**Comment by RuiAAPeres** [comment]
Thanks @NSMyself

**Pull request merged**